docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
+services:
+    scrud_db:
+      image: "mdillon/postgis:10"
```

Comment:
Do we need a GIS capable postgres? Would a vanilla postgres image suffice?

Reply:
wow you're right ... sorry ... I copied it from another file I have .. and I didn't see it was postgis ... I will replace that. thanks